recipes/serializable/meta.yaml

```diff
@@ -0,0 +1,43 @@
+{% set name = "serializable" %}
+{% set version = "0.1.1" %}
+{% set sha256 = "87f9fadbd0fba5c7951858d16ae9109afa4c96fd486e663419f3051f352a22d9" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  noarch: python
+  number: 0
+  script: python -m pip install --no-deps --ignore-installed .
+
+requirements:
+  build:
+    - python
+    - pip
+    - setuptools
+    - pypandoc
+  run:
+    - python
+    - typechecks >=0.0.2
+    - six >=1.9.0
+    - simplejson
+
+test:
+  imports:
+    - serializable
+
+about:
+  home: https://github.com/hamemrlab/serializable
```

Comment:
This url is dead, although the same link is on the pypi.io site. I can't find it on the hammerlab github organization either. 

@iskandr, @hammer -- do you know what the status of this project is and if there is a development repository somewhere that can be linked to?

Reply:
Just to be clear, there is a typo `hamemrlab` instead of `hammerlab` that appears on the pypi site. Searching the `hammerlab` organization on github, I can't find a repository named `serializable`.

Reply:
@npavlovikj - for now, can you change the `home` url to either:

https://pypi.org/project/serializable/

or probably:

https://github.com/iskandr/serializable

(just found Alex's repo for the project). We can finalize this in a pull request to the feedstock later on. Otherwise this PR looks good to me.

Reply:
Sorry I changed the project URL! Our former lab disbanded, I'll keep maintaining the package at https://github.com/iskandr/serializable.

Reply:
Thanks @iskandr . On a side note, I want to mention that none of your pypi packages have the Licence file in it. For now, I can pass the License file from the repo, but If you can add the license in the next pypi releases, that would be great and will make things much cleaner here.